arwen/contexts/storage.go

```diff
@@ -160,6 +156,11 @@ func (context *storageContext) GetStorageFromAddress(address []byte, key []byte)
 func (context *storageContext) getStorageFromAddressUnmetered(address []byte, key []byte) ([]byte, bool) {
 	var value []byte
 
+	if context.isElrondReservedKey(key) {
```

Comment:
I wonder if this needs the epoch flag as well. To be discussed.

Reply:
No need for flag.